src/iocore/net/Connection.cc

```diff
@@ -238,6 +241,23 @@ Server::setup_fd_for_listen(bool non_blocking, const NetProcessor::AcceptOptions
 #endif
   }
 
+#ifdef SO_INCOMING_CPU
+  if (opt.sockopt_flags & NetVCOptions::SOCK_OPT_INCOMING_CPU) {
+    int      cpu     = 0;
+    EThread *ethread = this_ethread();
+
+#ifdef TS_USE_HWLOC
+    cpu = ethread->hwloc_obj->os_index;
```

Comment:
The `hwloc_obj::os_index` looks "OS-provided physical index number", but hwloc has any better API?
https://github.com/open-mpi/hwloc/blob/63a8288d31a1baf67a909466aba9a022c78ca7b1/include/hwloc.h#L491